pkg/compiler/compiler.go

```diff
@@ -72,15 +69,26 @@ func (c *Compiler) Compile() error {
 		}
 		log := zap.L().With(zap.String("plugin", p.Name()))
 		log.Debug("starting")
-		links, err := p.Translate(c.Document.Constructs, c.Document.CloudResources)
+		links, err := p.Translate(c.Document.Constructs, c.Document.Resources)
 		if err != nil {
 			return core.NewPluginError(p.Name(), err)
 		}
 		c.Document.Configuration.AddLinks(links)
 		log.Debug("completed")
 	}
 
-	return nil
+	for _, p := range c.IaCPlugins {
+		if isPluginNil(p) {
```

Comment:
I know we did this before, so this is just a side commentary- Do we really need to be defensive about this? You'd have to go fairly out of your way to add a typed nil to the plugins.

Reply:
Jordan and I were confused by this, too! I'll make a note to nix it in the next round of cleanups.